src/v/pandaproxy/rest/proxy.cc

```diff
@@ -9,52 +9,125 @@
 
 #include "pandaproxy/rest/proxy.h"
 
+#include "config/property.h"
+#include "config/rest_authn_endpoint.h"
```

Comment:
Is this include needed?

Reply:
Yeah, the property is so I can create a binding bool and the endpoint is so I can check authn_type